### PR TITLE
Hotfix: saves Christmas

### PR DESCRIPTION
**Pull request by Horatio22**
## About The Pull Request

hotfixes devil removal by adding them back in as admin-spawn mobs

## Why It's Good For The Game
Slig wants it for buses. Merry Christmas, Slig

## Changelog

:cl:
add: Adds true devil, arch devil, and imps back in as simple mobs

/:cl:

### CODE REVIEW

**Comment by Horatio22** [comment]
Apologies for all the commits; I don't know how to code

File: tgstation.dme [hunk 1]
```diff
@@ -4094,6 +4094,7 @@
 #include "fulp_modules\features\mentors\mentorsay.dm"
 #include "fulp_modules\features\mentors\mentorwho.dm"
 #include "fulp_modules\features\mob_spawners\code\mobspawner.dm"
+#include "fulp_modules\features\mobs\fulp_mobs.dm"
```
Comment (SgtHunk) [inline tgstation.dme:4097]: Rename file to devil.dm plz

File: fulp_modules/features/mobs/fulp_mobs.dm [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	desc = "A large, menacing creature covered in armored black scales."
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "imp"
+	icon_living = "imp"
+	speak_emote = list("cackles")
+	health = 150
+	maxHealth = 150
+	obj_damage = 15
+	melee_damage_lower = 10
+	melee_damage_upper = 15
```
Comment (SgtHunk) [inline fulp_modules/features/mobs/fulp_mobs.dm:39]: Already exists as  
 ​/​mob​/​living​/​simple_animal​/​hostile​/​imp

**Review by SgtHunk — CHANGES_REQUESTED** [review]

**Comment by SgtHunk** [comment]
this got speedmerged wtf

**Comment by wraith-54321** [comment]
badmins speedmerging

**Pull request merged**